README.md

```diff
@@ -73,17 +81,25 @@ TBD
 
 TBD
 
-## Verifying Releases
+## Verifying Integrity
 
-Bomctl releases can be [found here](https://github.com/bomctl/bomctl/releases) and are signed
+### Verifying Container Images
+
+Bomctl container images can be [found here](https://hub.docker.com/repository/docker/bomctl/bomctl/general) and are signed
 using keyless signing with cosign.
 
-For each artifact there are two additional files:
+You can then verify this container image with cosign.
 
-- `${artifact}-keyless.sig` - signature
-- `${artifact}-keyless.pem` - certificate
+``` shell
+cosign verify --certificate-oidc-issuer https://token.actions.githubusercontent.com --certificate-identity-regexp 'https://github\.com/bomctl/bomctl/\.github/.+'  bomctl/bomctl:latest
+```
+
+### Verifying Releases
+
+Bomctl releases can be [found here](https://github.com/bomctl/bomctl/releases) and are signed
```

Comment:
Suggest reordering to allow lowercasing of `bomctl` or just surround with backticks if starting a sentence with it.

```suggestion
`bomctl` releases can be found [here](https://github.com/bomctl/bomctl/releases) and are signed
```